webapp/sources/rudder/rudder-rest/src/main/scala/com/normation/rudder/rest/lift/TechniqueApi.scala

```diff
@@ -235,7 +235,16 @@ class TechniqueApi(
           force <- extractBoolean("force")(req).map(_.getOrElse(false)).toIO
           _     <- techniqueWriter.deleteTechnique(techniqueInfo._1, techniqueInfo._2, force, modId, authzToken.qc)
         } yield {
-          Json.Obj(("id", Json.Str(techniqueInfo._1)), ("version" -> Json.Str(techniqueInfo._2)))
+          // here, up to 8.3 we used to just return: `"data": { "techniques": { "id":...}}`
```

Comment:
this is the part where we build by hand the return `data` object since we want to have an array or not depending of API version